nifi-nar-bundles/nifi-hubspot-bundle/nifi-hubspot-processors/src/main/resources/docs/org.apache.nifi.processors.hubspot.GetHubSpot/additionalDetails.html

```diff
@@ -0,0 +1,143 @@
+<ul>
+    <li>Access
+        <ul>
+            <li>Access Scope: none</li>
+            <li>StoreFront Access Token: none</li>
+        </ul>
+    </li>
+    <li>Analytics
+        <ul>
+            <li>Reports: updated_at_min</li>
+        </ul>
+    </li>
+    <li>Billing
+        <ul>
+            <li>Application Charge: none</li>
+            <li>Application Credit: none</li>
+            <li>Recurring Application Charge: none</li>
+        </ul>
+    </li>
+    <li>Customers
+        <ul>
+            <li>Customers: updated_at_min</li>
+            <li>Customer Saved Searches: none</li>
+        </ul>
+    </li>
+    <li>Discounts
+        <ul>
+            <li>Price Rules: updated_at_min</li>
+        </ul>
+    </li>
+    <li>Events
+        <ul>
+            <li>Events: created_at_min</li>
+        </ul>
+    </li>
+    <li>Inventory
+        <ul>
+            <li>Inventory Levels: updated_at_min</li>
+            <li>Locations: none</li>
+        </ul>
+    </li>
+    <li>Marketing Event
+        <ul>
+            <li>Marketing Events: none</li>
+        </ul>
+    </li>
+    <li>Metafields
+        <ul>
+            <li>Metafields: updated_at_min</li>
+        </ul>
+    </li>
+    <li>Online Store
+        <ul>
+            <li>Blogs: none</li>
+            <li>Comment: none</li>
+            <li>Pages: none</li>
+            <li>Redirects: none</li>
+            <li>Script Tags: updated_at_min</li>
+            <li>Themes: none</li>
+        </ul>
+    </li>
+    <li>Orders
+        <ul>
+            <li>Abandoned Checkouts: updated_at_min</li>
+            <li>Draft Orders: updated_at_min</li>
+            <li>Orders: updated_at_min</li>
+        </ul>
+    </li>
+    <li>Plus
+        <ul>
+            <li>Gift Cards: none</li>
+            <li>Users: none</li>
+        </ul>
+    </li>
+    <li>Product
+        <ul>
+            <li>Collects: none</li>
+            <li>Custom Collections: updated_at_min</li>
+            <li>Products: updated_at_min</li>
+            <li>Smart Collections: updated_at_min</li>
+        </ul>
+    </li>
+    <li>Sales Channels
+        <ul>
+            <li>Collection Listings: none</li>
+            <li>Mobile Platform Applications: none</li>
+            <li>Product Listings: updated_at_min</li>
+            <li>Resource Feedbacks: none</li>
+        </ul>
+    </li>
+    <li>Shipping and Fulfillments
+        <ul>
+            <li>Carrier Services: none</li>
+        </ul>
+    </li>
+    <li>Store Properties
+        <ul>
+            <li>Countries: none</li>
+            <li>Currencies: none</li>
+            <li>Policies: none</li>
+            <li>Shipping Zones: updated_at_min</li>
+            <li>Shop: none</li>
+        </ul>
+    </li>
+    <li>Tender Transactions
+        <ul>
+            <li>Tender Transactions: processed_at_min</li>
+        </ul>
+    </li>
+</ul>
+
+Specific trap type can set in case of Enterprise Specific generic trap type is chosen.
```

Comment:
Does this comment apply to this processor, or should it be removed?

Reply:
It should be removed, thanks!